server/grpc/api/ownership.go

```diff
@@ -3,26 +3,32 @@ package api
 import (
 	"context"
 
+	"github.com/mesg-foundation/engine/cosmos"
+	ownershippb "github.com/mesg-foundation/engine/ownership"
 	protobuf_api "github.com/mesg-foundation/engine/protobuf/api"
 	"github.com/mesg-foundation/engine/sdk"
+	"github.com/mesg-foundation/engine/x/ownership"
 )
 
 // OwnershipServer is the type to aggregate all Ownership APIs.
 type OwnershipServer struct {
-	sdk *sdk.SDK
+	sdk    *sdk.SDK
+	client *cosmos.Client
 }
 
 // NewOwnershipServer creates a new OwnershipServer.
-func NewOwnershipServer(sdk *sdk.SDK) *OwnershipServer {
-	return &OwnershipServer{sdk: sdk}
+func NewOwnershipServer(sdk *sdk.SDK, client *cosmos.Client) *OwnershipServer {
+	return &OwnershipServer{
+		sdk:    sdk,
+		client: client,
+	}
 }
 
 // List returns all ownerships.
 func (s *OwnershipServer) List(ctx context.Context, req *protobuf_api.ListOwnershipRequest) (*protobuf_api.ListOwnershipResponse, error) {
-	ownerships, err := s.sdk.Ownership.List()
-	if err != nil {
+	var ownerships []*ownershippb.Ownership
+	if err := s.client.QueryJSON("custom/"+ownership.ModuleName+"/"+ownership.QueryListOwnerships, nil, &ownerships); err != nil {
```

Comment:
As said in other comment, we should use specific constant as much as possible. Here, replace `ModuleName` by `QuerierRoute`.
```suggestion
	if err := s.client.QueryJSON("custom/"+ownership.QuerierRoute +"/"+ownership.QueryListOwnerships, nil, &ownerships); err != nil {
```